.pre-commit-config.yaml

```diff
@@ -7,27 +7,13 @@ ci:
 
 repos:
 -   repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: 'v0.12.2'
+    rev: 'v0.12.7'
     hooks:
+    # Run the linter:
     -   id: ruff
         name: ruff
-        additional_dependencies:
```

Comment:
why are additional deps not needed?

Reply:
I'm not sure why it was added, it works without them